src/popup/modules/InitQrCode.js

```diff
@@ -85,3 +85,38 @@ export const initiationProcess = Promise.all([qrCreatorInit, userInterfaceInit])
 }).catch((error) => {
     Logger.logError(error);
 });
+
+/**
+ * Queries Tabs API for 20 times with a delay of 300ms until the tabs have a defined url.
+ * Will reject if url is not defined.
+ * @returns {Promise}
+ */
+async function getsTabWithValidUrl() {
+    let retryCount = 0;
+    const maxRetries = 20;
+    const delay = 300;
```

Comment:
As these are constants and literals you may write them `CONSTANT_STYLE`. But here they are not in a module scope, but in a function (as it is explained in the contributing doc), so I am not sure here… May think about this later… :thinking:

Reply:
I won't consider these application level constants, they are just used inside this function. I could have just pass the numbers to retryPromise without assining them first to a variable.

Reply:
Yeah, I had the same thought. So I guess it is okay.